deploy/deployment_dev_cluster.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: image.openshift.io/v1
```

Comment:
can't we have the same manifest for "local" and "dev cluster" deployments? we could even take advantage of this for local testing: just push a new image to the container registry and get the application redeployed with the DeployConfig + ImageStream combination.

Reply:
ok, let's keep this PR as short as possible :)

Reply:
@xcoulon, we can't use the same setup for development, as service needs to be created as `NodePort`, Also, imagePullPolicy is different (`IfNotPresent` -> for minishift to work with `eval $(minishift docker-env)` and `Always` for production environment.

We can have some script to generate this on fly, but as we are getting rid out of this, it's not needed